src/pages-and-resources/discussions/app-list/FeaturesTable.scss

```diff
@@ -0,0 +1,4 @@
+table {
```

Comment:
that's very generic & will match every table element.

Reply:
Yes. It's generic but i will impact only on `FeatureTable.jsx` because I import `FeatureTable.scss` file only for  `FeatureTable.jsx`